packages/freezed/test/integration/multiple_constructors.dart

```diff
@@ -38,3 +38,10 @@ abstract class Recursive with _$Recursive {
   factory Recursive() = RecursiveImpl;
   factory Recursive.next(RecursiveImpl value) = _RecursiveNext;
 }
+
+@freezed
+abstract class RequiredParams with _$RequiredParams {
+  const factory RequiredParams({@required String a}) = RequiredParams0;
+  const factory RequiredParams.second({@required String a}) = RequiredParams1;
+  const factory RequiredParams.third({@required String b}) = RequiredParams2;
```

Comment:
Is there any purpose to the `third` constructor?

Reply:
No :-) will remove